test/e2e/models/CodeServer.ts

```diff
@@ -30,15 +31,18 @@ export class CodeServer {
     // but usually a reload or two fixes it
     // TODO@jsjoeio @oxy look into Firefox reconnection/timeout issues
     while (!editorIsVisible) {
+      // When a reload happens, we want to wait for all resources to be
+      // loaded completely. Hence why we use that instead of DOMContentLoaded
+      // Read more: https://thisthat.dev/dom-content-loaded-vs-load/
+      await this.page.waitForLoadState("load")
+      // Give it an extra second just in case it's feeling extra slow
+      await this.page.waitForTimeout(1000)
```

Comment:
ahah these things make me sad, totally fair though

Reply:
lol i know. sadly, this timeout was the only thing that made it _consistently_ pass.